internal/pkg/plan/persist_action.go

```diff
@@ -8,31 +8,30 @@ import (
 
 type PersistAction interface {
 	String() string
+	Path() string
 }
 
 type NewConfigAction struct {
-	Key         model.ConfigKey
-	Path        string
```

Comment:
`PersistAction`s maju historicky fieldy `Path` a `ProjectPath`, ... co bolo neskor na inych miestach nahradene za `PathInProject` ... no tu sa na to zabudlo.

Tak to v tomto PR zjednocujem, aby nam to nekomplikovalo pracu.